### PR TITLE
Add consul dns check

### PR DESCRIPTION
**Pull request by stefanschneider**
[#108474830]

### CODE REVIEW

**Comment by stefanschneider** [comment]
Story link: https://www.pivotaltracker.com/story/show/108474830

File: src/hakim/hakim.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 
 var requiredProcesses = []string{"consul.exe", "containerizer.exe", "garden-windows.exe", "rep.exe", "metron.exe"}
 
+var bbsCosulHost = "bbs2.service.cf.internal."
```
Comment (mhoran) [inline src/hakim/hakim.go]: What is this host? On our deployment, bbs is at `bbs.service.cf.internal`, not `bbs2`.

Comment (stefanschneider) [inline src/hakim/hakim.go]: Thx. did the wrong commit after some testing.

**Comment by benmoss** [comment]
Fixed this up and committed in 66c3620251e5ce713cec3c993b7d308a2a6898cc. Thanks Stefan!

**Pull request closed without merging**